lte/gateway/docker/services/c/Dockerfile

```diff
@@ -104,9 +104,8 @@ RUN apt-get update && apt-get install -y \
 ENV MAGMA_ROOT /magma
 WORKDIR /magma
 
-# Copy Bazel files at root and third_party
-COPY WORKSPACE.bazel BUILD.bazel .bazelignore .bazelrc .bazelversion ${MAGMA_ROOT}/
-COPY bazel/ ${MAGMA_ROOT}/bazel
+COPY . ${MAGMA_ROOT}
+COPY lte/gateway/docker/mme/configs/ ${MAGMA_ROOT}/lte/gateway/docker/configs/
```

Comment:
This configs are moved manually - is this important for this ci flow or should they be moved in the repository?